Do not override MacPorts optimization flags.
https://github.com/outpaddling/endian/commit/2f35a052c0ae404e016c2ac418818aaa40b54bca

Fix manpage install directory.
https://github.com/outpaddling/endian/pull/1
--- Makefile.orig	2020-05-15 19:19:18.000000000 -0500
+++ Makefile	2024-07-19 22:09:26.000000000 -0500
@@ -1,6 +1,6 @@
 OBJS	= endian.o
 CC	?= gcc
-CFLAGS	+= -Wall -O
+CFLAGS	?= -Wall -O
 PREFIX	?= /usr/local
 
 all:	endian
@@ -19,6 +19,6 @@
 
 install: endian
 	mkdir -p ${STAGEDIR}${PREFIX}/bin \
-		${STAGEDIR}${PREFIX}/man/man1
+		${STAGEDIR}${PREFIX}/share/man/man1
 	install -c -m 0755 endian ${STAGEDIR}${PREFIX}/bin
-	install -c -m 0644 endian.1 ${STAGEDIR}${PREFIX}/man/man1
+	install -c -m 0644 endian.1 ${STAGEDIR}${PREFIX}/share/man/man1
